yaqc/yaqc/_socket.py

```diff
@@ -8,40 +8,54 @@
 import fastavro  # type: ignore
 
 from ._schema import handshake_request, handshake_response
+from ._logging import logger
 
 
 BUFFSIZE = 4096
 
 
 class Socket:
     def __init__(self, host, port):
+        self._host = host
+        self._port = port
         self._socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         self._socket.settimeout(None)
         self._socket.connect((host, port))
         self._named_types = {}
 
     def _read(self, response_schema):
+        logger.info(
+            f"{self._host}:{self._port} Socket._read response_schema {response_schema}"
+        )
+        logger.info(f"{self._host}:{self._port} Socket._read checkpoint A")
         response_schema = fastavro.parse_schema(
             response_schema, expand=True, named_schemas=self._named_types
         )
         # Needed twice for nested types, likely should be fixed upstream
+        logger.info(f"{self._host}:{self._port} Socket._read checkpoint B")
         response_schema = fastavro.parse_schema(
             response_schema, expand=True, named_schemas=self._named_types
         )
         buf = io.BytesIO()
         remaining = 0
+        logger.info(f"{self._host}:{self._port} Socket._read checkpoint C")
         while True:
             try:
+                logger.info(f"{self._host}:{self._port} Socket._read try {buf}")
```

Comment:
```suggestion
                logger.info(f"{self._host}:{self._port} Socket._read try {buf.getvalue()[:64}}")
```

repr/str for a BytesIO object is not helpful, truncated for readability/not logging really long things. Nothing in the suspect area SHOULD be that long, sooo